test/v1alpha1/clients.go

```diff
@@ -97,3 +102,20 @@ func newClients(t *testing.T, configPath, clusterName, namespace string) *client
 	c.ConditionClient = cs.TektonV1alpha1().Conditions(namespace)
 	return c
 }
+
+// getEmbeddedStatusValue gets the current value for the "embedded-status" feature flag.
+// If the flag is not set, it returns the default value. This is used in tests which look
+// for something in the "TaskRuns", "Runs", or "ChildReferences" fields of a PipelineRun's
+// status to ensure that those tests check the right things depending on the value of
+// "embedded-status".
+func getEmbeddedStatusValue(ctx context.Context, t *testing.T, c *clients) string {
```

Comment:
this function is duplicated, can it be reused instead?

Reply:
Annoyingly? No. Because `test.v1alpha1.clients` is a different struct than `test.clients`. Though I could change that to pass `kubeClient kubernetes.Interface` instead of `c *clients` - I'll do that.

Reply:
that sounds much better!